js/index.js

```diff
@@ -1,79 +1,78 @@
-let books = [];
-const booksElement = document.querySelector('.books');
-
-function BookModel(title, author) {
-  this.title = title;
-  this.author = author;
-}
-
-function saveData() {
-  localStorage.setItem('dataBooks', JSON.stringify(books));
-}
-
-function createBook(book) {
-  const bookElement = document.createElement('div');
-  bookElement.classList.add('book');
-
-  const titleElement = document.createElement('h2');
-  titleElement.classList.add('title');
-  titleElement.textContent = book.title;
-
-  const authorElement = document.createElement('h3');
-  authorElement.classList.add('author');
-  authorElement.textContent = book.author;
-
-  const removeButton = document.createElement('button');
-  removeButton.classList.add('remove');
-  removeButton.textContent = 'Remove';
-  removeButton.addEventListener('click', () => {
-    removeBook(book);  // eslint-disable-line
-  });
+const booksContainer = document.querySelector('.books');
+const form = document.getElementById('addBook');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
 
-  const hrTag = document.createElement('hr');
+class BookList {
+  constructor() {
+    this.books = [];
+  }
 
-  bookElement.appendChild(titleElement);
-  bookElement.appendChild(authorElement);
-  bookElement.appendChild(removeButton);
-  bookElement.appendChild(hrTag);
+  addBook(book) {
+    if (localStorage.getItem('localData') === null) {
+      localStorage.setItem('localData', JSON.stringify([]));
+      this.books.push(book);
+      localStorage.setItem('localData', JSON.stringify(this.books));
+      window.location = window.location.pathname;
+    } else {
+      this.books = JSON.parse(localStorage.getItem('localData'));
+      this.books.push(book);
+      localStorage.setItem('localData', JSON.stringify(this.books));
+      window.location = window.location.pathname;
+    }
+  }
 
-  return bookElement;
-}
+  removeBook(title) {
+    this.books = JSON.parse(localStorage.getItem('localData'));
+    this.books.forEach((book, index) => {
+      if (book.title === title) {
+        this.books.splice(index, 1);
```

Comment:
[OPTIONAL} Consider using the filter method provided by JavaScript in order to update the array in an immutable way